graal-sdk/build.gradle

```diff
@@ -44,9 +44,18 @@ publishing {
                 // Details about the artifact being published
                 groupId = "com.r3.conclave"
                 artifactId = "graal-sdk"
-                version = "$graal_version-$conclave_version"
+                version = getConclaveGraalVersion()
                 description = 'Graal SDK'
             }
+
+            pom {
+                licenses {
+                    license {
+                        name = 'Universal Permissive License, Version 1.0'
+                        url = 'http://opensource.org/licenses/UPL'
```

Comment:
Not sure I understand, what is this license? Isn't the original Graal provided with a GPL + Classpath exception?
If I understand correctly, we only can use 2 licenses
1 - full GPL
2 - GPL with classpath exception

Reply:
GraalVM has different licenses each its components. Have a look at its README: https://github.com/oracle/graal#license.

This artifact is UPL, as evidenced by the published artifact from Oracle: https://repo1.maven.org/maven2/org/graalvm/sdk/graal-sdk/22.2.0/graal-sdk-22.2.0.pom

Reply:
@filipesoliveira this question from Marco shows that we need to have a comment here.

Reply:
Interesting thanks, so it looks fine. It seems that that license WAS actually GPL+CPE and then they changed it https://github.com/oracle/graal/commit/c90742063beaeed879a29b65eca6abe3930146a6
So with "Graal SDK" I guess they mean "that bunch of configs that assemble things together"?

Reply:
No, The Graal SDK is the Java API to use GraalVM features from the java code that will be compiled with GraalVM. Confusing, I know!

Reply:
Yep, ok, did not gig further. Sounds interesting anyway how they think that the GPL+CPE does not infect this layer.